opentelemetry/src/main/scala/zio/telemetry/opentelemetry/tracing/ErrorMapper.scala

```diff
@@ -2,11 +2,11 @@ package zio.telemetry.opentelemetry.tracing
 
 import io.opentelemetry.api.trace.StatusCode
 
-case class ErrorMapper[E](body: PartialFunction[E, StatusCode])
+case class ErrorMapper[E](body: PartialFunction[E, StatusCode], toThrowable: Option[E => Throwable])
```

Comment:
I think it would be better to make `toThrowable` equal to None by default.
```
case class ErrorMapper[E](body: PartialFunction[E, StatusCode], toThrowable: Option[E => Throwable] = None)
```

Reply:
fixed